src/index.js

```diff
@@ -84,7 +102,12 @@ const processFilter = (filterQS, castFn, preprocessor) => {
     if (cast) query = `(${preprocessed})::${cast}`;
   }
 
-  return `${query} ${conditionMap[condition]}`;
+  const currCondition = conditionMap[condition];
+  if (currCondition.includes('??')) {
+    return currCondition.replace('??', query);
```

Comment:
just making sure this is equivalent with return `${query} ${currCondition}`; in line 110?

Reply:
Yea, on line 110 we prepend `query`, but if the query includes ?? we instead replace `query` wherever the `??` is.